src/App.js

```diff
@@ -1,10 +1,43 @@
+import { useEffect, useState } from 'react';
 import './App.css';
 import Calculator from './components/Calculator';
+import Qoutes from './components/Qoutes';
 
 function App() {
+  const [data, setData] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    const category = 'learning';
+    const url = `https://api.api-ninjas.com/v1/quotes?category=${category}`;
+    const myKey = 'AUj4hMwj4zZWOLjZ1Z9NOQ==F4PGbcfkNTtQOc23';
```

Comment:
Exposing API keys directly in code, as shown in the example const `myKey = 'AUj4hMwj4zZWOLjZ1Z9NOQ==F4PGbcfkNTtQOc23';`, is not considered best practice from a security standpoint. This approach could potentially expose sensitive information if the code is ever shared or exposed unintentionally.

A better practice is to use environment variables to store sensitive information like API keys. By utilizing environment variables, you can keep sensitive data separate from your codebase and ensure that it's not directly visible in the source code. 

When working with APIs or other external services, it's recommended to use environment variables to store keys, tokens, and other sensitive information. This approach helps protect your application and data, reducing the risk of accidental exposure.

To use environment variables in a React application, you can follow these steps:

1. **Create an Environment File:**

   Start by creating a `.env` file in the root of your React project. You can have different versions of this file for different environments (development, production, etc.). For example, you can create a `.env.development` file for development.

2. **Set Environment Variables:**

   In the `.env` file, you can set your environment variables in the format `REACT_APP_VARIABLE_NAME=value`. Remember to prefix your variable names with `REACT_APP_` to ensure they are recognized as environment variables by Create React App.

   Example `.env.development`:
   ```
   REACT_APP_API_KEY=your_api_key_here
   REACT_APP_BASE_URL=https://api.example.com
   ```

3. **Access Environment Variables in Your Code:**

   In your React components or other parts of your code, you can access the environment variables using `process.env.REACT_APP_VARIABLE_NAME`.

   ```jsx
   import React from 'react';

   function App() {
     const apiKey = process.env.REACT_APP_API_KEY;
     const baseUrl = process.env.REACT_APP_BASE_URL;

     // Use apiKey and baseUrl in your code
     // ...

     return (
       // Your component JSX
     );
   }

   export default App;
   ```

4. **Using Environment Variables in Build:**

   When you build your React app (e.g., for production), the environment variables you've set will be replaced with their actual values. You don't need to manually change them for different environments. Just ensure that you have the appropriate `.env` files for each environment.

5. **Important Note:**

   Keep in mind that while environment variables are useful for managing sensitive data, they are still accessible to anyone with access to the compiled JavaScript code. If your application requires extremely sensitive information, consider using a server to fetch and provide the data to the client securely.

Additionally, if you are using Create React App, there's no need to install any specific package for handling environment variables, as it comes with built-in support for them. Just follow the steps above, and you're good to go.